packages/renderer/src/components/organisms/Sidebar/__snapshots__/Sidebar.test.tsx.snap

```diff
@@ -304,7 +302,7 @@ exports[`molecules::Sidebar > renders without exploding 1`] = `
                       <div
                         class="mantine-Text-root mantine-14d5cdm"
                       >
-                        14.048212174412
+                        NaN
```

Comment:
This indicates an error in your logic. The UI should never display error values like `NaN`